exe/clockwork-lint

```diff
@@ -10,6 +10,7 @@ class ClockworkWorkersList
   def initialize(config_store)
     @workers = Set.new
     file_path = 'config/clockwork.rb'
+    exit unless File.exist?(file_path)
```

Comment:
I think, it's not polite to `exit` right from the initialize. Why don't you put this check outside?

Reply:
No good reason. Was just going with the smallest possible change.